### DISCUSSION TITLE
RPC URL from MetaMask

### DISCUSSION THREAD

**Discussion by lotinasa** [wevm/wagmi#2117] [Q&A]
Is it possible to configure wagmi providers in such way to use primarily MetaMask RPC URL and then if it fails to use some of the fallbacks (alchemy, infura, jsonRpcProvider)?

In wagmi configureChains I can set static RPC URLs and predefined alchemy/infura RPC URLs, but is it possible to use dynamic RPC URL from wallet and then from configuration?

**Comment by rsodre** [comment]
I'd love that too!
